tests/php/validation/test-class-amp-validation-manager.php

```diff
@@ -581,7 +581,6 @@ public function test_override_validation_error_statuses_with_good_nonce() {
 	 * Test validation error overrides for when bad nonce is supplied.
 	 *
 	 * @covers AMP_Validation_Manager::override_validation_error_statuses()
-	 * @expectedException WPDieException
```

Comment:
@dhaval-parekh Are these annotations no longer supported?

Reply:
This annotation is deprecated and will be removed in PHPUnit 9.